### PR TITLE
fix: Incorrect description of customizing babel

### PR DESCRIPTION
**Pull request by rkostrzewski**
How come no one has experienced this and no issue was raised yet? :laughing:

### CODE REVIEW

**Comment by rkostrzewski** [comment]
Cancelled build that's why it shows as red.

**Comment by lukeed** [comment]
How about:

---

To customize Babel, you have two options:

1. You may create a [`.babelrc`](https://babeljs.io/docs/usage/babelrc/) file in your project's root directory. Any settings you define here will overwrite matching config-keys within [Preact CLI preset]. For example, if you pass a `"plugins"` object, it will replace & reset all Babel plugins that Preact CLI defaults to.

2. If you'd like to modify or add to the existing Babel config, please see the [Webpack](#webpack) section and customize via `preact.config.js`:

    ```js
    // preact.config.js
    export default (config, env, helpers) => {
      let { rule } = helpers.getLoadersByName(config, 'babel-loader');
      rule.options.plugins.push('my-chosen-plugin');
    };
    ```

**Comment by thangngoc89** [comment]
@rkostrzewski Tips: add [skip ci] in commit message to prevent CI build. 

Not related but shallow merge is weird here.

**Comment by rkostrzewski** [comment]
@lukeed +1 - I'm writing a recipe for that on the wiki and will link there
@thangngoc89 yeah I've thought to myself before commiting `gotta remember to add [skip ci]` 😆

File: README.md [hunk 1]
```diff
@@ -134,7 +134,12 @@ By default, `preact-cli` emulates the following config:
 ```
 
 #### Babel
-To customize babel simply create [```.babelrc```](https://babeljs.io/docs/usage/babelrc/) file. [Preact CLI preset] will be applied automatically so you won't have to worry about keeping your ```.babelrc``` updated!
+
+To customize Babel, you have two options:
+
+1. You may create a [`.babelrc`](https://babeljs.io/docs/usage/babelrc/) file in your project's root directory. Any settings you define here will overwrite matching config-keys within [Preact CLI preset]. For example, if you pass a `"plugins"` object, it will replace & reset all Babel plugins that Preact CLI defaults to.
+
+2. If you'd like to modify or add to the existing Babel config, `preact.config.js` must be used (see the [Webpack](#webpack) section for reference) - [here's an example](https://github.com/developit/preact-cli/wiki/Config-Recipes#customising-babel-options-using-loader-helpers) on how to achieve that.
```
Comment (lukeed) [inline README.md]: ...existing Babel config, you must use a `preact.config.js` file. Visit the [Webpack](#section) for more info, or check out the [Customize Babel](https://github.com/developit/preact-cli/wiki/Config-Recipes#customising-babel-options-using-loader-helpers) example!

Comment (rkostrzewski) [inline README.md]: Thanks! Have you ever considered a career as a technical writer? 😆

**Pull request merged**